spec/services/doc_auth/mock/result_response_builder_spec.rb

```diff
@@ -268,5 +268,47 @@
         expect(response.pii_from_doc).to eq({})
       end
     end
+
+    context 'with a yaml file containing integer zipcode' do
+      let(:input) do
+        <<~YAML
+          document:
+            first_name: Susan
+            last_name: Smith
+            middle_name: Q
+            address1: 1 Microsoft Way
+            address2: Apt 3
+            city: Bayside
+            state: NY
+            zipcode: 11364
```

Comment:
should we add a test for an integer zipcode that starts with a 0? Then YAML will parse it as octal which gives weird results. I feel like this should be an error but unsure

```ruby
YAML.load <<-STR
zipcode: 01234
STR
=> {"zipcode"=>668}